crates/algokit_transact/src/transactions/mod.rs

```diff
@@ -116,6 +121,12 @@ impl AssetTransferTransactionBuilder {
     }
 }
 
+impl KeyRegistrationTransactionBuilder {
```

Comment:
I noticed we have all the builders defined here. Probably makes sense to move them into their respective files?